### PR TITLE
Switch gh-pages to rc2

### PR DESCRIPTION
**Pull request by yohanboniface**
To be merged when we have actually released and rc2 links are available on npmcdn.

### CODE REVIEW

**Comment by mourner** [comment]
Are we fine using https JS links on a http site? Any browsers that would block this kind of mixed requests?

**Comment by Eschon** [comment]
This is only a problem when it is the other way around (http resources on a https page).
[According to Paul Irish](http://www.paulirish.com/2010/the-protocol-relative-url/) you should always use https resources when they are available.

File: _posts/2016-07-16-leaflet-1.0-rc2.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+---
+layout: post
+title: Announcing Leaflet 1.0-rc2
+description: Leaflet 1.0 Release Candidate 2 is out
+author: Yohan Boniface
+authorsite: http://yohanboniface.me
+---
+
+Here comes the second release candidate for Leaflet 1.0,  the more stable Leaflet version ever!
```
Comment (jieter) [inline _posts/2016-07-16-leaflet-1.0-rc2.md]: 'the more' sounds a bit odd, did you intend that?

File: _posts/2016-07-16-leaflet-1.0-rc2.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+
+Leaflet 1.0-rc2 is mainly bug fixing, with two notable exceptions. The full list of changes is in the [1.0.0-rc2 changelog](https://github.com/Leaflet/Leaflet/blob/master/CHANGELOG.md#10-rc2-july-18-2016) as usual.
+
+
+### Notable changes
+
+* Event refactoring: thanks to an initial work from [@fab1an](https://github.com/fab1an), coordinated by [@perliedman](https://github.com/perliedman), the events are greatly optimized: they are faster, and consume less memory.
+
+* New `L.Tooltip` class: started as a port of Leaflet.Label plugin to Leaflet core, and then reworked, this new class allows to display small tooltips attached to map features. The API is very similar to what Leaflet.Label was (with some exceptions in options naming and default values), so the upgrade path from Leaflet.Label to `L.Tooltip` should be straightforward, and we encourage you to do so while upgrading your Leaflet to 1.0.
```
Comment (jieter) [inline _posts/2016-07-16-leaflet-1.0-rc2.md]: Might be nice to describe this upgrade path in a little more detail, maybe just here.

File: _posts/2016-07-16-leaflet-1.0-rc2.md (hunk 1 above)
Comment (yohanboniface) [inline _posts/2016-07-16-leaflet-1.0-rc2.md]: If the question is "did I intend the sentence to be odd", the answer is no :p
But you know, my English… ;)
How would you express that, then? `stablest` seems a bit odd to me too.

File: _posts/2016-07-16-leaflet-1.0-rc2.md (hunk 2 above)
Comment (yohanboniface) [inline _posts/2016-07-16-leaflet-1.0-rc2.md]: Yeah, I though about that too, but thought that the best place for that would be the Leaflet.Label README? What do you think?

File: _posts/2016-07-16-leaflet-1.0-rc2.md (hunk 1 above)
Comment (jieter) [inline _posts/2016-07-16-leaflet-1.0-rc2.md]: I would say 'the most stable Leaflet verstion ever!' I guess.

File: _posts/2016-07-16-leaflet-1.0-rc2.md (hunk 2 above)
Comment (jieter) [inline _posts/2016-07-16-leaflet-1.0-rc2.md]: Yeah, Leaflet.label readme sounds good to me, I'd say it's good to have that in place before releasing this blog post, before any confusion can come to existence.

Comment (yohanboniface) [inline _posts/2016-07-16-leaflet-1.0-rc2.md]: Added to https://github.com/Leaflet/Leaflet/issues/4732, and I'll do a PR to Leaflet.Label for that.

Comment (jieter) [inline _posts/2016-07-16-leaflet-1.0-rc2.md]: I think we should add a link to the new leaflet.label readme here.

**Pull request merged**